packages/matchbox/src/helpers/props.ts

```diff
@@ -9,11 +10,11 @@
  * omit({ my: '100', className: 'classs' }, margin.propNames)
  *  > { className: 'class' }
  */
-export function omit(props, names) {
+export function omit(props: { [k: string]: any }, names: string[]): { [k: string]: any } {
```

Comment:
shoudl we use unknown instead of any?

Reply:
Yeah, I tried any it was breaking a few files